lib/sync-server.js

```diff
@@ -100,7 +100,12 @@ var DEFAULT_SYNC_CONF = {
   /**@type {String} specify the maximum retention time of an inactive datasetClient. Any inactive datasetClient that is older than this period of time will be removed.*/
   datasetClientCleanerRetentionPeriod: '24h',
   /** @type {String} specify the frequency the datasetClient cleaner should run. Default every hour.*/
-  datasetClientCleanerCheckFrequency: '1h'
+  datasetClientCleanerCheckFrequency: '1h',
+
+  /** @type {Boolean} Specify if the server should wait for the ack insert to complete before returning the response for the sync request. Default is true. */
+  syncReqWaitForAck: true,
+  /** @type {Number} Specify the max number of ack items will be processed for a single request. Default is -1 (unlimited).*/
+  syncReqAckLimit: -1
```

Comment:
[Question] I might miss a lots of context here but what will happen if some acknowledges will be dropped by using this value? Is this going to drop them?
Can we assume that we going to save just x first ack from customer? 
Do they need to be sorted/ordered? 

This seem to be really low level option and I'm not sure when users may change that without harming their data.